packages/core/src/application.ts

```diff
@@ -51,6 +57,7 @@ export class Application extends Context {
     if (!options) options = {};
 
     this.bind('http.port').to(options.http ? options.http.port : 3000);
+    this.api({basePath: '/', paths: {}});
```

Comment:
I wonder if the method name should be `app.rest(...)`. Later on, we might have other mappings, such as `app.grpc(...)`.